lib/dotcom_web/live/trip_planner.ex

```diff
@@ -106,6 +106,15 @@ defmodule DotcomWeb.Live.TripPlanner do
     {:noreply, new_socket}
   end
 
+  @impl true
+  # Triggered when we cannot connect to OTP.
+  def handle_async("get_itinerary_groups", {:ok, {:error, %Req.TransportError{reason: :econnrefused}}}, socket) do
+    message = "Cannot connect to OpenTripPlanner. Please try again later."
+    new_socket = assign(socket, :results, Map.put(@state.results, :error, message))
+
+    {:noreply, new_socket}
+  end
```

Comment:
📓 No ticket for this, I just noticed that if OTP is unreachable, we didn't have a way to handle it.